docs/meshstack.building-pipeline-integration.md

```diff
@@ -0,0 +1,61 @@
+---
+id: meshstack.building-pipeline-integration
+title: Connecting meshStack and a Pipeline
+---
+
+In this section you may find information about how to:
+
+1. Trigger a pipeline via meshStack to deploy and delete resources
+2. Update users on the progress of an external automation 
+
+## Trigger a Pipeline
+
+Please find below the steps that are required to trigger an external automation via meshStack.
+
+### 1. Preparation
+
+At the moment we don't provide a native integration into any CI/CD tooling but we offer a neat way to trigger external systems via Building Blocks. In order to trigger a pipeline you may use the Terraform Building Block template below as a place to start. The module below can be used to trigger Github actions, we call this module "Trigger Module".
+
+The strucutre of the module is as follows:
+
+- [**main.tf**](./assets/terraform_examples/pipeline_main): Calls webhooks that are used to trigger Github actions via terraform apply and terraform destroy
+- [**variables.tf**](./assets/terraform_examples/pipeline_var): Variables necessary for the integration with GitHub
+
+Host your version of the module in a repository of your choice.
+
+### 2. Creating the Building Block Definition
+
+In order to provide an existing automation you already got covered in a pipeline triggered by meshStack you need to prepare a Building Block Definition. We advise to create this Definition in one of the workspaces that have "service management" enabled. Make sure to use the Building Block Definition of type Terraform/OpenTofu and provide the information for the repository hosting your version of the "Trigger Module".
```

Comment:
t: actually this is the only way to create it inside a workspace via "service management". Later you need an API token for sending the updates and such API tokens can only  be scoped to user workspaces. If a BB def would be created as an admin, you would not be able to create an API token which is scoped to the admin/partner workspace.